src/EloquentHashids.php

```diff
@@ -51,4 +51,12 @@ public static function getHashidEncodingValue(Model $model)
   {
     return $model->getKey();
   }
+  
+  /**
+   * @return string
+   */
+  public function getRouteKeyName()
+  {
+      return 'uid';
```

Comment:
Ohhhh! Yes, of course. I'll change it on the PR. Hope it's accepted some day. :)